cpp/examples/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ add_executable(ide_secir_example ide_secir.cpp)
 target_link_libraries(ide_secir_example PRIVATE memilio ide_secir)
 target_compile_options(ide_secir_example PRIVATE ${MEMILIO_CXX_FLAGS_ENABLE_WARNING_ERRORS})
 
+add_executable(history_example history_example.cpp)
+target_link_libraries(history_example PRIVATE memilio)
+target_compile_options(history_example PRIVATE ${MEMILIO_CXX_FLAGS_ENABLE_WARNING_ERRORS})

```

Comment:
I think we should we stick to the usual scheme of adding the "_example" only to the cmake target, and not the filename. Since the .cpp file is already in the examples directory, it is not necessary to put example in its name again.

Reply:
true!